code/unclassified/src/leap_year/leap_year.java

```diff
@@ -0,0 +1,24 @@
+public class LeapYear {
+	public static boolean isLeapYear(int year) {
+		if(year % 4 == 0)
+			return true;
+		if(year % 400 == 0)
+			return true;
+		if(year % 100 == 0)
+			return false;
+		
+		return false;
+	}
+	public static void main(String []args) {
+		Scanner in = new Scanner(new InputStreamReader(System.in));
+		System.out.println("Enter starting year");
```

Comment:
Input should be "Enter the starting year: "
Other one should be "Enter the ending year: "